.github/scripts/handle-mochawesome-report.js

```diff
@@ -0,0 +1,114 @@
+const fs = require("fs");
+
+const logger = { debug: console.debug, error: console.error };
+
+const collectTestsByType = ({ type, suite, path, cache = [] }) => {
+  logger.debug(`Collecting tests of type ${type}`);
+  const localCache = cache;
```

Comment:
Mutating an argument is not great but since it's a small standalone script... :see_no_evil: